Userland/DevTools/UserspaceEmulator/main.cpp

```diff
@@ -43,15 +44,17 @@ int main(int argc, char** argv, char** env)
     if (dump_profile && profile_instruction_interval == 0)
         profile_instruction_interval = 128;
 
-    DeprecatedString executable_path;
-    if (arguments[0].contains("/"sv))
-        executable_path = Core::DeprecatedFile::real_path_for(arguments[0]);
-    else
-        executable_path = Core::DeprecatedFile::resolve_executable_from_environment(arguments[0]).value_or({});
-    if (executable_path.is_empty()) {
+    ErrorOr<String> executable_path_or_error { EINVAL }; // Dummy value
+    if (arguments[0].contains("/"sv)) {
+        executable_path_or_error = FileSystem::real_path(arguments[0]);
+    } else {
+        executable_path_or_error = FileSystem::resolve_executable_from_environment(arguments[0]);
+    }
```

Comment:
Suggestion:
```cpp
auto executable_path_or_error = arguments[0].contains('/')
    ? FileSystem::real_path(arguments[0])
    : FileSystem::resolve_executable_from_environment(arguments[0]);
```

Reply:
Normally I don't like ternaries, but here it avoids the dummy value nicely. Thanks!